swagger/api/swagger/swagger.yaml

```diff
@@ -15,16 +15,23 @@ produces:
   - application/json
   - application/x-protobuf
   - image/png
+tags:
+  - name: wxTiles
+    description: wxTiles API
+  
```

Comment:
I removed all the tags and replaced them with a single wxTiles tags so the generated code is nicer.